examples/src/main/scala/zio/config/examples/CoproductExample.scala

```diff
@@ -45,7 +45,7 @@ object CoproductExample extends App {
   val anotherSource: ConfigSource =
     mapSource(validConfigForAnotherConfig)
 
-  assert(runtime.unsafeRun(read(prodOrDev).run.provide(anotherSource))._2 == Right(Dev("v3", 1, 2.0)))
+  assert(runtime.unsafeRun(read(prodOrDev).provide(anotherSource))._2 == Right(Dev("v3", 1, 2.0)))
```

Comment:
👍

Reply:
Let's revert the moving of lines of code and deletion of files.
We can do it later. Let this use case be solving one issue of having the return types to be just ZIO